JavaScriptSPA/authConfig.js

```diff
@@ -1,21 +1,25 @@
+
+// Config object to be passed to Msal on creation.
+// For a full list of msal.js configuration parameters, 
+// visit https://azuread.github.io/microsoft-authentication-library-for-js/docs/msal/modules/_authenticationparameters_.html
 const msalConfig = {
-    auth: {
-        clientId: "e760cab2-b9a1-4c0d-86fb-ff7084abd902",
-        authority: "https://fabrikamb2c.b2clogin.com/fabrikamb2c.onmicrosoft.com/b2c_1_susi",
-        validateAuthority: false
-    },
-    cache: {
-        cacheLocation: "localStorage",
-        storeAuthStateInCookie: true
-    }
+  auth: {
+    clientId: "e760cab2-b9a1-4c0d-86fb-ff7084abd902",
+    authority: "https://fabrikamb2c.b2clogin.com/fabrikamb2c.onmicrosoft.com/b2c_1_susi",
+    validateAuthority: false
+  },
+  cache: {
+    cacheLocation: "localStorage", // This configures where your cache will be stored
+    storeAuthStateInCookie: true // Set this to "true" if you are having issues on IE11 or Edge
```

Comment:
Change the comment to "Please set this to true to save cache in cookies to address trusted zones limitations in IE" - and add the corresponding documentation in MSAL wiki.

Reply:
I've set the default value to false as well (that's the recommendation, right?)